### PR TITLE
Ng/up and running with recoil a new state management library for react

### PR DESCRIPTION
**Pull request by NicollGuarnizo**
TIme stamps for https://egghead.io/courses/up-and-running-with-recoil-a-new-state-management-library-for-react-78b8

### CODE REVIEW

File: up-and-runnig-with-recoil-a-new-state-management-library-for-react/01-use-recoil-atom-to-share-state-between-two-react-components.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+# Set up environment 
+
+The documentation on how to set up the latest version of react can be found [here](https://reactjs.org/docs/create-a-new-react-app.html)
+
+1. Inside of an empty directory/folder run the following command to create a new react project `npx create-react-app running-w-recoil` 
+2. Run `cd running-w-recoil`
+3. Finally, to run your react server you could either run `npm start` or `yarn start`
+
+<TimeStamp start="0:00" end="0:02">
```
Comment (joelhooks) [inline up-and-runnig-with-recoil-a-new-state-management-library-for-react/01-use-recoil-atom-to-share-state-between-two-react-components.md:11]: this is a big ol' note and it's set to display for only 2 seconds

I think we want to generally think about how the notes will display to the user and they will popup and close based on the start/end times.

File: up-and-runnig-with-recoil-a-new-state-management-library-for-react/03-use-selectors-to-calculate-derived-data-based-on-state-stored-within-a-recoil-atom.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Notes
+
+<TimeStamp start="0:27" end="0:29">
```
Comment (joelhooks) [inline up-and-runnig-with-recoil-a-new-state-management-library-for-react/03-use-selectors-to-calculate-derived-data-based-on-state-stored-within-a-recoil-atom.md:3]: this is also short in duration

"how long does this take to read and process" is the general guideline here.

**Comment by joelhooks** [comment]
looks great! just had a couple comments about short times on the notes, but otherwise this looks good to me.

File: up-and-runnig-with-recoil-a-new-state-management-library-for-react/01-use-recoil-atom-to-share-state-between-two-react-components.md (hunk 1 above)
Comment (NicollG) [reply up-and-runnig-with-recoil-a-new-state-management-library-for-react/01-use-recoil-atom-to-share-state-between-two-react-components.md:11]: I could leave this note to show up for 7 seconds, or just not include it in the timestamp as this was not explained by the instructor, it was more like an extra note for the learner. What do you think @joelhooks?

File: up-and-runnig-with-recoil-a-new-state-management-library-for-react/03-use-selectors-to-calculate-derived-data-based-on-state-stored-within-a-recoil-atom.md (hunk 2 above)
Comment (zacjones93) [reply up-and-runnig-with-recoil-a-new-state-management-library-for-react/03-use-selectors-to-calculate-derived-data-based-on-state-stored-within-a-recoil-atom.md:3]: Would it be possible to set a minimum duration for notes displaying with a reasonable default?

**Review by laurosilvacom — APPROVED** [review]

**Pull request merged**